### PR TITLE
fix String#pluralize issue10

### PR DESCRIPTION
**Pull request by balmoral**
Still learning the git fork-branch-pr workflow so apologies if not following protocols correctly.

Was not able to run formal tests - all the string inflection tests were commented out - but manual testing passed.

### CODE REVIEW

**Comment by mibamur** [comment]
@balmoral fix this error `https://travis-ci.org/opal/opal-activesupport/builds/134629302#L176`
write test would be great

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+.idea
```
Comment (elia) [inline .gitignore]: this line should be [added to your global `.gitignore` file](http://stackoverflow.com/questions/7335420/global-git-ignore#7335487)

Comment (elia) [inline .gitignore]: Also you should `git rm --cached .idea make`

**Comment by elia** [comment]
Closing, as #10 has been fixed by #15

**Pull request closed without merging**